src/pages/FishIndex.js

```diff
@@ -1,11 +1,23 @@
 import React, { Component } from 'react'
+import { NavLink } from 'react-router-dom'
+// import { Card, CardTitle, Col } from 'reactstrap'
```

Comment:
If these imports are not being used they can be removed.

Reply:
fixing in progress